docs/src/pages/careers/react-engineer-core.md

```diff
@@ -0,0 +1,126 @@
+They are working on 4 products (Material UI, MUI Base, Joy UI, and MUI System) with a tiny team.
+
+We also need help to continue to improve the health of the open-source product: make the core components easier to use, engage more with the community, polish all the details, make the components more accessible, guiding developers to answers, and just generally being a positive presence in the open-source community.
+
+## About the role
+
+### Why this is interesting
+
+Our solution empowers React developers to build awesome applications faster – we see over a million developers on MUI's docs every month. – hundreds of thousands of developers use MUI every month.
+
+The core components are massively used, you will receive a lot of feedback from your work. The community has high expectations, they will push you to do better, every day.
+
+### What you'll do on a day-to-day basis
+
+Depending on the day, you'll:
+
+- **Help guide architectural decisions**.
+  The future of MUI is discussed and planned in our public RFCs and issues.
+  You'll be helping drive these conversations and guiding MUI toward the best possible solutions.
+- **Contribute to deep, meaningful refactors and feature releases**.
+  MUI is a complex codebase. Components we've shipped recently, such as the data grid have required months of dedicated, careful work.
+- **Reduce friction**.
+  A large amount of the work on MUI is reducing friction and making it easier to use.
+  This might involve careful API design, identifying and fixing top bugs, creating easier to understand error messages, and writing documentation and blog posts about features you ship.
+- **Collaborate with the community**.
+  Many small as well as meaningful fixes and features have been contributed by the community. Your role is to draw the best out of the community — to inspire those across the world to create and contribute through your reviews of their issues and pull requests.
+- **Experiment and play**. Great, unexpected features and heisenbug fixes have come from a number of sources — relentlessly methodical processes of elimination, free-flowing team collaboration, inspiration by adjacent libraries and projects, and difficult-to-explain individual strokes of brilliance. Whatever your preferred style is for creating new things that others might not have thought of, you'll find a welcome home on the team.
+- **Take ownership of features from idea/mockup to live deployment**.
+  You'll shape and guide the direction of crucial new features, including new components.
+- **Ship. Early and often**. You'll iterate and ship frequently.
+  You'll have a real impact on the end-user experience and you'll love working on a team that builds stunning UIs and prioritizes delivering real user value as often as possible.
+- **You'll be interacting with the users** on a regular basis, handling inbound support and feature requests (every developer helps with developer requests).
```

Comment:
I think we specifically add something related to a11y? Or maybe as part of the "Skills you should have" section

Reply:
Feel free to iterate on it